service/defaultcomponents/defaults.go

```diff
@@ -90,7 +90,7 @@ func Components() (
 		jaegerexporter.NewFactory(),
 		fileexporter.NewFactory(),
 		otlpexporter.NewFactory(),
-		kafkaexporter.NewFactory(),
+		kafkaexporter.NewFactory(kafkaexporter.DefaultMarshallers()),
```

Comment:
Can the marshaler be optional?

Reply:
Done in the next commit, the CI failure does not seem to be related to this PR.